src/test/java/org/wmn4j/io/musicxml/MusicXmlFileChecks.java

```diff
@@ -0,0 +1,713 @@
+package org.wmn4j.io.musicxml;
+
+import org.wmn4j.notation.elements.*;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class MusicXmlFileChecks {
```

Comment:
This class looks like it’s not supposed to be instantiated. If a class is not meant to be instantiated, then it can be made non-instantiable by adding an empty private constructor. This file could also have the license header comment at the top.

Reply:
Good tip, I changed that!

Reply:
Also added the license and Javadoc for the class.